Modules/CTP/src/RawDataQcTask.cxx

```diff
@@ -46,6 +46,8 @@ void CTPRawDataReaderTask::initialize(o2::framework::InitContext& /*ctx*/)
   getObjectsManager()->startPublishing(mHistoBC);
   getObjectsManager()->startPublishing(mHistoInputs);
   getObjectsManager()->startPublishing(mHistoClasses);
+  mDecoder.setDoLumi(1);
```

Comment:
These two methods take a boolean, please don't use `0` and `1` but use a boolean.

Reply:
Ok, thanks! Should I change it now, or in the next release is ok? :-)

Reply:
next release is ok